frodo.js

```diff
@@ -9,46 +9,48 @@
  * Copyright 2016 Henrik Ingo (@henrikingo)
```

Comment:
SCRIPT-Well done with the efficient error handling, it covers all the possible scenarios.